README.md

```diff
@@ -11,100 +11,25 @@ This library comes in two parts:
 
     Verify a quote issued by an SGX/TDX TEE
 
-This repo provides a simple Go library to enable these features.
+This repo provides a simple Go library to enable these features and is used by [Constellation](https://github.com/edgelesssys/constellation) to enable TDX attestation.
 
-## Quote Verification
+## Restrictions
+- The current version is based on [DCAP 1.15](https://github.com/intel/SGXDataCenterAttestationPrimitives/releases/tag/DCAP_1.15). 
+and has been tested on the kernel and libraries from the tdx-tools release [2023ww01](https://github.com/intel/tdx-tools/releases/tag/2023ww01).
+Given that the UAPI for TDX is yet to be finished and upstreamed, newer versions might not work yet.
```

Comment:
```suggestion
- The current version is based on [DCAP 1.15](https://github.com/intel/SGXDataCenterAttestationPrimitives/releases/tag/DCAP_1.15),
and has been tested on the kernel and libraries from the tdx-tools release [2023ww01](https://github.com/intel/tdx-tools/releases/tag/2023ww01).
Given that the UAPI for TDX is yet to be finished and upstreamed, newer versions might not be supported.
```

Reply:
I made the first one a separate sentence.